scoring.js

```diff
@@ -0,0 +1,83 @@
+const attempts = 0
+const completions = 0
+const passingYards = 1 / 25
+const touchdowns = 6
+const interceptions = -3
+const rushingYards = 1 / 10
+const fumbles = -3
+const receptions = 1
+const receivingYards = 1 / 10
+const returnYards = 1 / 15
+const returns = 0
```

Comment:
We should never have variables outside of functions, we talked about that a few week's back.